torchvision/models/regnet.py

```diff
@@ -587,6 +587,21 @@ class RegNet_Y_16GF_Weights(WeightsEnum):
             "acc@5": 98.054,
         },
     )
+    IMAGENET1K_SWAG_LINEAR_V1 = Weights(
```

Comment:
I'm OK with this name. Should we rename `IMAGENET1K_SWAG_V1` to `IMAGENET1K_SWAG_E2E_V1` to indicate that there was end-to-end finetuning on top of the original swag weights? No strong opinions.

Reply:
I agree with this, and will change it